core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/deletion/WorkspaceDeletionRunnerSpec.scala

```diff
@@ -184,6 +184,32 @@ class WorkspaceDeletionRunnerSpec extends AnyFlatSpec with MockitoSugar with Mat
     }
   }
 
+  it should "return complete when leo returns a 404 for the workspace when deleting apps" in {
```

Comment:
Am I misunderstanding this test? It appears to be verifying that the step returns `Incomplete` not `Complete`

Reply:
I just named the test poorly (and somewhat robotically).
The actually important part is that it moves on to the next step. 
I'll fix the wording.